docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -17,12 +13,12 @@ This feature is supported only in 3.0.0-RC1.
      - [Sample directory structure](#sample-directory-structure)
    - [Prerequisites for all domain types](#prerequisites-for-all-domain-types)
    - [Additional prerequisites for JRF domains](#additional-prerequisites-for-jrf-domains)
-   - [Initial use case](#initial-use-case): An initial WebLogic domain
-   - [Update1 use case](#update1-use-case): Dynamically adding a data source using a model ConfigMap
-   - [Update2 use case](#update2-use-case): Deploying an additional domain
-   - [Update3 use case](#update3-use-case): Updating an application in an image
-   - [Cleanup](#cleanup)
-   - [References](#references)
+   - [Initial](#initial-use-case) use case: An initial WebLogic domain
+     - [Update 1]({{< relref "/samples/simple/domains/model-in-image/update1.md" >}}): Dynamically adding a data source using a model ConfigMap
+     - [Update 2]({{< relref "/samples/simple/domains/model-in-image/update2.md" >}}): Deploying an additional domain
+     - [Update 3]({{< relref "/samples/simple/domains/model-in-image/update3.md" >}}): Updating an application in an image
```

Comment:
I prefer 'Update1' to 'Update 1'.  It's just as understandable/readable IMOO, it conveniently makes the doc easier to search/edit for a specific use case - no need to use two different search terms, plus it lines up with the internal documentation/scripts for same.

Reply:
Personally, I don't like the titles Update 1, Update 2 and such. Perhaps, we can think of a more descriptive heading for each updated use case. That might solve our dilemma.